tests/main.fmf

```diff
@@ -0,0 +1,23 @@
+require:
+    - make
+    - python3-pip
+
+/lint:
+    tag: [ upstream ]
+    summary: Run Linting on the whole codebase
+    test: cd .. && make lint
+
+/integration:
+    tag: [ upstream, downstream ]
+    summary: Run integration tests
+    test: cd .. && make integration
+
+/unittest:
+    tag: [ upstream, downstream ]
```

Comment:
do you plan to run unittest downstream?

Reply:
no. and again, this is coverage of tests, so it's not even a test run. it's good to measure how much the codebase is covered